downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/Main.java

```diff
@@ -0,0 +1,219 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+
+public class Main {
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void main(String[] args) throws Exception {
+		
+		logger.info("Beginning Download Directory step");
+		String pathToConfig = "src/main/resources/config.properties";
```

Comment:
This probably should be configurable, in case the properties file is on a different path.